particles/Native/Wasm/BUILD

```diff
@@ -17,29 +17,18 @@ cc_wasm_binary(
     deps = ["//src/wasm/cpp:arcs"],
 )
 
-arcs_manifest(
+arcs_kt_schema(
     name = "wasm_schemas",
-    srcs = ["Harness.arcs"],
+    src = "Harness.arcs",
```

Comment:
What's the justification for this restriction? Most other similar rules in blaze/bazel take a list of sources (e.g. proto_library). If you were to have a large directory of schemas, instead of using file globing, e.g. srcs = glob(["*.arcs"]), you'd have to write one arcs_kt_schema rule per file.

Reply:
Done